core/src/main/java/org/mule/runtime/core/processor/ReferenceProcessor.java

```diff
@@ -47,6 +48,11 @@ public Event process(Event event) throws MuleException {
     return referencedProcessor.process(event);
   }
 
+  @Override
+  public Publisher<Event> apply(Publisher<Event> publisher) {
```

Comment:
Dan it is used by the process method which is delgated to the underalying (referencedProcessor)